bin/wetzel.js

```diff
@@ -5,12 +5,14 @@ var path = require('path');
 var argv = require('minimist')(process.argv.slice(2), { boolean : ["w", "suppresswarnings" ]});
 var defined = require('../lib/defined');
 var defaultValue = require('../lib/defaultValue');
-var generateMarkdown = require('../');
+var enums = require('../lib/enums');
+var generateMarkdown = require('../lib/generateMarkdown');
 
 if (!defined(argv._[0]) || defined(argv.h) || defined(argv.help)) {
     var help = 'Usage: node ' + path.basename(__filename) + ' [path-to-json-schema-file] [OPTIONS]\n' +
         '  -l,  --headerLevel        Top-level header. Default: 1\n' +
         '  -p,  --schemaPath         The path string that should be used when generating the schema reference paths.\n' +
+        '  -al, --autoLink           Aggressively auto-inter-link types referenced in descriptions.  Add =cqo to auto-link types that are in code-quotes only.\n' +
```

Comment:
Seems like `-al` is not working, possibly because it's 2 letters?

* `--autoLink` and `--autoLink=cqo` both work fine (as long as I remember that L is uppercase).
* `--al` and `--al=cqo` both work.
* `-al` is ignored safely.
* `-al=cqo` does something strange, removing all of the `###` section header indicators.  For example:

```
$ bin/wetzel.js ../glTF/specification/2.0/schema/glTF.schema.json -al | grep camera\.extensions
### camera.extensions

$ bin/wetzel.js ../glTF/specification/2.0/schema/glTF.schema.json -al=cqo | grep camera\.extensions
 camera.extensions
```